chainer/links/normalization/layer_normalization.py

```diff
@@ -0,0 +1,53 @@
+from chainer import initializers
+from chainer import link
+
+from chainer.functions.array import broadcast
+from chainer.functions.math import sqrt
+from chainer.functions.math import square
+from chainer.functions.math import sum
+from chainer.links.connection import scale
+
+
+class LayerNormalization(link.Chain):
+
+    """Layer normalization layer on outputs of linear functions.
+
+    This is a link of "Layer Normalization". This layer
+    normalizes input units by statistics along the second axis,
+    scales and shifts them with :class:`~chainer.links.Scale`.
+
+    Args:
+        size (int): Size of input units.
```

Comment:
Is size required in here in `__init__()`? Consider using the "parameter shapeholder" feature so that size can be removed from here.